components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/API.java

```diff
@@ -651,6 +652,15 @@ public String getApiLevelPolicy() {
     public void setApiLevelPolicy(String apiLevelPolicy) {
         this.apiLevelPolicy = apiLevelPolicy;
     }
-    
-    
+
+    public boolean isWS() {
+        return isWebsocketApi;
+    }
+
+    public void setWS(String WS) {
+        if("ws".equalsIgnoreCase(WS)||"true".equalsIgnoreCase(WS))
```

Comment:
use String constant called WS and use simple latter for variable names. we use camel case for variable names. Also there is a formatting issue where you need space before and after ||. Also it bit confuse to check ws".equalsIgnoreCase(WS) may be you can use better name fovariable name